VMWare/vmware-esxi/tests/AUTH_SSH_process_opened.json

```diff
@@ -0,0 +1,37 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "vmware-esxi",
+        "dialect_uuid": "2b13307b-7439-4973-900a-2b58303cac90"
+      }
+    },
+    "message": "info hostd[2099655] [Originator@6876 sub=Vimsvc.ha-eventmgr] Event 974676 : SSH session was opened for 'root@1.2.3.4'."
+  },
+  "expected": {
+    "message": "info hostd[2099655] [Originator@6876 sub=Vimsvc.ha-eventmgr] Event 974676 : SSH session was opened for 'root@1.2.3.4'.",
+    "observer": {
+      "vendor": "VMware",
+      "product": "ESXi"
+    },
+    "event": {
+      "kind": "event",
+      "category": [
+        "process",
+        "session"
+      ],
+      "type": [
+        "info"
```

Comment:
For this event, a more suitable event.type will be ["start"] as it represent the starting of a new session.